demo/src/features/debug3d/write.rs

```diff
@@ -66,14 +144,15 @@ impl FeatureCommandWriter for Debug3dCommandWriter {
                 command_buffer.cmd_draw(draw_call.count as u32, draw_call.first_element as u32)?;
             }
         }
+
         Ok(())
     }
 
     fn feature_debug_name(&self) -> &'static str {
-        Debug3dRenderFeature::feature_debug_name()
+        render_feature_debug_name()
```

Comment:
As a matter of style I would write `super::render_feature_debug_name()` (or `CrateName::function()` if it's wasn't in super) to make it clear to a reader where the function is from.